MAINTAINERS.md

```diff
@@ -0,0 +1,197 @@
+candidate should create a PR for adding their key to the repository and to the
+above table (see https://github.com/bitcoin/bitcoin/pull/23798 for example).
+During / Before the PR, regular contributors are encouraged to correspond with
+candidate(s) publicly about any concerns or questions. Candidates are also
+encouraged to write up a Prioritization statement (see below) in conjunction
+with the key PR.
+
+At the discretion of the maintainers, and no sooner than 2 weeks after the PR
+is opened, the candidate(s) PR may be merged.
+
+
+## Maintainer Prioritizations
+
+While the maintainers are ultimately not accountable to any particular roadmap,
+nor can make regular contributors focus on one area or another, practically the
+views and prioritizations of maintainers have an impact on what gets reviewed,
+merged, and the general direction of the project.
+
+As such, maintainers should regularly deliver a written "state of the union"
+article on what they intend to focus on in the coming period. Such addresses
+should be done per-release cycle, and should minimally cover:
+
+1. What went well in the last release cycle
+1. What is making good progress for this release cycle
+1. How the maintainer intends to prioritize outstanding
+   PRs/issues/features/projects in the next release cycle
+1. Projects which could use participation from other contributors
+1. Longer term focuses beyond the next release
+
+Maintainers are generally free to structure their posts as they like. Posts are
+not expected to be "all new content", they can be simple updates from the last
+post. Posts may also be more frequent, if there is significant updates to share.
+
+Historical examples of similar postings:
+
+- [@laanwj](https://laanwj.github.io/2021/01/21/decentralize.html)
+- [@achow101](https://achow101.com/2020/10/0.21-wallets)
+
+
+Below, each maintainer should keep up-to-date links of current and past
+prioritization addresses (as well as any other useful materials in that
+respect).
+
+#### @achow101
+#### @fanquake
+#### @hebasto
+#### @laanwj
+#### @marcofalke
+#### @sipa
+
+Non maintainers may also share similar documents detailing their work here:
+
+
+## Maintainers "Moving On"
+
+Maintainers may choose to cease being a maintainer at any time without notice.
+Where possible, maintainers should try to gracefully transition out of their
+role, until a new maintainer has been selected to replace them (if needed).
+
+## Maintainer Removal
+
+
+Maintainers should be removed by other maintainers if there is any reason to
+believe the maintainer or their keys may be compromised -- this removal can be
+done temporarily by any single maintainer, unless there are only three
+remaining maintainers (in which case removal requires 2, a majority), but
+should be unanimously decided if permanent removal. While imperfect, this
+procedure ensures that access can be revoked quickly in the event of a
+compromise, while preventing takeover by a single dev.
```

Comment:
> preventing takeover by a single dev

But it allows takeover by 2 cooperating maintainers. These hard rules have the downside that they make it clear to adversaries what exactly has to be done in order to f**k up the project.

Reply:
Well right now we "allow", afaict, takeover by a single dev it's just not clearly stated.

Is security through obscurity preferred?